lib/ykpiv.c

```diff
@@ -128,6 +130,34 @@ ykpiv_rc ykpiv_disconnect(ykpiv_state *state) {
   return YKPIV_OK;
 }
 
+static ykpiv_rc _select_application(ykpiv_state *state) {
```

Comment:
The other static functions in the file have bare names, so please change this to select_application()